Sources/SwiftDoc/Interface.swift

```diff
@@ -55,9 +55,9 @@ public final class Interface: Codable {
     public private(set) lazy var relationships: [Relationship] = {
         var relationships: Set<Relationship> = []
         for symbol in symbols {
-            let `extension` = symbol.context.compactMap({ $0 as? Extension }).first
+            let owner = symbol.context.last(where: { $0 is Extension || $0 is Symbol })
```

Comment:
I'm not sure `owner` the right name for this. Can we come up with something better? (If not, we can go with something inelegant but precise, like `lastExtensionOrSymbolContext`)

Reply:
I can also propose `lastDeclarationScope`, what do you think?

Reply:
Perfect!